app/javascript/mastodon/components/dropdown_menu.js

```diff
@@ -43,6 +44,10 @@ class DropdownMenu extends React.PureComponent {
     this.dropdown.hide();
   }
 
+  handleShow = () => this.setState({ expanded: true })
+
+  handleHide = () => this.setState({ expanded: false })
+
```

Comment:
That's neat; I didn't know you could skip the explicit `.bind()` by using arrow functions here, but apparently [you can](https://babeljs.io/repl/#?babili=false&evaluate=true&lineWrap=false&presets=es2015%2Creact%2Cstage-2&targets=&browsers=&builtIns=false&debug=false&code=class%20Foo%20%7B%0A%20stringify%20%3D%20()%20%3D%3E%20this.toString()%0A%7D). Very cool!